ern-util/package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "bluebird": "^3.5.0",
     "chalk": "^2.3.0",
-    "code-push": "^2.0.3-beta",
+    "code-push": "^2.0.4",
```

Comment:
wondering if we can move to v2.1.4?

Reply:
There is no `v2.1.4` I think they did a mistake and meant `2.0.4`

Reply:
After verification. 2.1.4 is the version of the CLI, not the SDK ... That's confusing ;P

Reply:
So we're all good.